RecoLocalCalo/HcalRecProducers/src/MahiGPU.cu

```diff
@@ -1100,6 +1112,7 @@ namespace hcal {
           outputGPU.recHits.timeM0.get(),
           outputGPU.recHits.did.get(),
           totalChannels,
+	  conditions.channelQuality.status,
```

Comment:
this seems like an odd choice by the code-format

Reply:
it has a tab instead of eight spaces